code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1594,6 +1594,7 @@
 		dna.real_name = real_name
 
 	species.handle_post_spawn(src)
+	species.handle_dna(src) //Give them whatever special dna business they got.

```

Comment:
Oh, if you're adding this line, could you then also remove [this bit](https://github.com/ParadiseSS13/Paradise/blob/61da53cca1b7841838a212a6e856a1033d281634/code/modules/mob/living/carbon/human/human.dm#L52-L53), so that we don't do the same code multiple times? (since human `New` calls `set_species` anyways, and the species' special DNA stuff should apply before the DNA gets sync'd to the limbs, anyways)